doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+        * An application-specific storage, enabled with the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_CUSTOM` option.
+
+ * :ref:`lib_nrf_cloud_agps` library:
+
+    * Fixed premature assistance suppression when the :kconfig:option:`CONFIG_NRF_CLOUD_AGPS_FILTERED` option is enabled.
+      Added a 10 minute margin of error to ensure A-GPS assistance is downloaded every two hours even if the modem requests assistance a little early.
+
+* Removed:
+
+  * Cloud API library.
+
+Libraries for NFC
+-----------------
+
+* Updated:
+
+  * :ref:`nfc_ndef_parser_readme`:
+
+    * Updated:
+
+      * :c:func:`nfc_ndef_msg_parse` with a fix to the declaration, a new assertion to avoid a potential usage fault, and added a note in the function documentation.
+      * ``NFC_NDEF_PARSER_REQUIRED_MEMO_SIZE_CALC`` macro has been renamed to :c:macro:`NFC_NDEF_PARSER_REQUIRED_MEM`.
+
+Other libraries
+---------------
+
+* Added:
+
+  * :ref:`event_manager_proxy` library.
+  * :ref:`QoS` library.
+  * :ref:`emds_readme` library.
+  * :ref:`lib_dfu_multi_image` library.
+
+
+* Updated:
+
+  * :ref:`app_event_manager`:
+
+    * Added:
+
+      * Event type flags to represent if event type should be logged, traced, and whether it has dynamic data.
+        To update your application, pass a flag variable as a parameter in :c:macro:`APP_EVENT_TYPE_DEFINE` instead of ``init_log``.
+        Use :c:macro:`APP_EVENT_FLAGS_CREATE` to set multiple flags:
+
+        .. code-block:: c
+
+           APP_EVENT_TYPE_DEFINE(my_event,
+             log_my_event,
+             &my_event_info,
+             APP_EVENT_FLAGS_CREATE(APP_EVENT_TYPE_FLAGS_1, APP_EVENT_TYPE_FLAGS_2));
+
+      * :c:func:`app_event_manager_event_size` function with corresponding :kconfig:option:`CONFIG_APP_EVENT_MANAGER_PROVIDE_EVENT_SIZE` option.
+
+      * Universal hooks for Application Event Manager initialization, event submission, preprocessing, and postprocessing.
+        This includes implementation of macros that register hooks, grouped as follows:
+
+        * :c:macro:`APP_EVENT_HOOK_ON_SUBMIT_REGISTER`, :c:macro:`APP_EVENT_HOOK_ON_SUBMIT_REGISTER_FIRST`,  :c:macro:`APP_EVENT_HOOK_ON_SUBMIT_REGISTER_LAST`
+        * :c:macro:`APP_EVENT_HOOK_PREPROCESS_REGISTER`, :c:macro:`APP_EVENT_HOOK_PREPROCESS_REGISTER_FIRST`, :c:macro:`APP_EVENT_HOOK_PREPROCESS_REGISTER_LAST`
+        * :c:macro:`APP_EVENT_HOOK_POSTPROCESS_REGISTER`, :c:macro:`APP_EVENT_HOOK_POSTPROCESS_REGISTER_FIRST`, :c:macro:`APP_EVENT_HOOK_POSTPROCESS_REGISTER_LAST`
+
+    * Updated:
+
+      * Renamed Event Manager to Application Event Manager.
+
+  * :ref:`app_event_manager_profiler_tracer`:
+
+    * Updated:
+
+      * The library is no longer directly referenced from the Application Event Manager.
+        Instead, it uses the Application Event Manager hooks to connect with the manager.
+
+
+  * :ref:`esb_readme`:
+
+    * Fixed a compilation error for nRF52833.
+
+  * Partition Manager:
```

Comment:
```suggestion
  * :ref:`lib_spm`:
```

Reply:
Actually, this should be :ref:`partition_manager` , lib_spm is Secure Partition Manager ;) Changed.